firestore/google/cloud/firestore_v1/watch.py

```diff
@@ -213,9 +213,9 @@ def __init__(
         self._closing = threading.Lock()
         self._closed = False
 
-        initial_request = firestore_pb2.ListenRequest(
-            database=self._firestore._database_string, add_target=self._targets
-        )
+        self.resume_token = None
+
+        initial_request = self._get_initial_request
```

Comment:
Nit: This is not really the initial request anymore. It's a builder for the initial and all following requests.

Reply:
I know :(